lib/chai/core/assertions.js

```diff
@@ -537,8 +537,8 @@ module.exports = function (chai, _) {
         itemsCount = val.length;
         break;
       case 'function':
-        var name = val.name ? ' ' + val.name : '';
-        throw new TypeError('.empty was passed a function' + name);
+        var msg = '.empty was passed a function ' + _.getName(val);
+        throw new TypeError(msg.trim());
```

Comment:
Ping to @lucasfcosta: I think perhaps we should call `.trim()` any time we return a value from `getName` - just a note for https://github.com/chaijs/check-error/pull/11 and https://github.com/chaijs/chai/pull/813